parsl-introduction.ipynb

```diff
@@ -443,7 +448,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# App that generates a random number\n",
+    "# App that generates a semi-random number between 0 and 32,767\n",
```

Comment:
pseudo-random?

Reply:
I'm not sure pseudo-random is correct, but maybe - the next person can update it further if they wish